main/src/com/google/refine/util/HttpClient.java

```diff
@@ -1,3 +1,35 @@
+/*
+
+Copyright 2010, Google Inc.
```

Comment:
We probably should not "Google Inc." as the copyright holder, but rather "OpenRefine contributors"

Reply:
okay thanks let me correct that.

Reply:
No that would not be appropriate. We should only add the "OpenRefine contributors" header to the files which do not have a header yet. The existing headers should be left as they are.

Reply:
Point noted thanks  Let me update it .